config.py

```diff
@@ -770,17 +770,25 @@ def __init__(self, directory='templates'):
         self.__load_templates()
 
     def __load_templates(self):
-        '''Loads template files if they exist, othwerwise it sets defaults'''
+        '''Look in environment variables for templates.  If not set in env,
+        load template files if they exist. Othwerwise it sets defaults'''
 
         for template in self.t:
             name = self.t[template].full_name
-            try:
-                filename = os.path.join(self.__template_directory, name)
-                with open(filename) as f:
-                    logger.info('overriding %s from %s', name, filename)
-                    self.t[template].value = f.read()
-            except IOError:
-                logger.debug("setting default value for %s", name)
+            if os.environ.get(name):
```

Comment:
Would it be possible to add a test to `tests/test_marathon_lb.py`? You should be able to do something like

```
os.environ["key"] = "val"
```